### ISSUE TITLE
`ConfigParser.write` takes 1 positional argument but 2 were given 

### ISSUE THREAD

**Issue by marinelay** [kivy/kivy#8678]
**Software Versions**
* Python: 3.9.18
* OS: Ubuntu 22.04
* Kivy: 2.3.0
* Kivy installation method: pip install

**Describe the bug**

https://github.com/kivy/kivy/blob/c0d4894384abb81cf4729984a1eaa4c437d266da/kivy/__init__.py#L482-L487

`Config` is object of `ConfigParser` and `ConfigParser.write` method does not require any positional arguments.
Thus, it always raises `TypeError: write() takes 1 positional argument but 2 were given` at line 484.
I believe setting `self.filename` as `kivy_config_fn` is more accurate like this:
```python
Config.filename = kivy_config_fn
Config.write()
```
I wonder it is the right behavior.

**Expected behavior**
```python
Config.filename = kivy_config_fn
Config.write()
```

**To Reproduce**
```python
from kivy.config import Config

with open("abc", 'w') as fd:
        Config.write(fd)
```

**Code and Logs and screenshots**
https://github.com/kivy/kivy/blob/c0d4894384abb81cf4729984a1eaa4c437d266da/kivy/__init__.py#L482-L487

**Comment by misl6** [comment]
Nice catch!

I can confirm the issue is reproducible with `python examples/demo/touchtracer/main.py -m monitor --save`
I guess no one used the `--save` option for the last  ~12 years? 😅

Would you like to make a PR for it?

**Comment by misl6** [comment]
Fixed via #8681
